csrankings.csv

```diff
@@ -2,6 +2,7 @@ name,affiliation,homepage,scholarid
 Xiaolei Huang,Pennsylvania State University,https://faculty.ist.psu.edu/suh972/,iTtzc1UAAAAJ
 Zihan Zhou,Pennsylvania State University,https://faculty.ist.psu.edu/zzhou/Home.html,vEcgp3AAAAAJ
 Xiang Zhang 0001,Pennsylvania State University,https://faculty.ist.psu.edu/xzz89/,NOSCHOLARPAGE
+A Min Tjoa,TU Wien,http://www.ifs.tuwien.ac.at/tjoa,x8qCMhcAAAAJ

```

Comment:
Not listed here: http://www.informatik.tuwien.ac.at/faculty/professors

Reply:
He is listed on the website mentioned. You can find him under 'T'. I have used the same website to include all current faculty members who can _solely_ advise PhD students.

Reply:
Somehow missed it - thanks.